Models/wallGoExampleBase.py

```diff
@@ -228,7 +227,7 @@ def runExample(self) -> None:  # pylint: disable = R0914, R0915
         collisionTensor: "WallGoCollision.CollisionTensor" | None = None
 
         # hacky
-        momentumGridSize = manager.config.getint("PolynomialGrid", "momentumGridSize")
+        momentumGridSize = manager.config.configGrid.momentumGridSize
 
```

Comment:
For the grid size, I'd suggest we make a getter function WallGoManager.getMomentumGridSize() or similar. Because this particular variable is needed by collision stuff too so good to expose it more explicitly